packages/react-ui/src/atoms/Typography/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
I don't know if that folder `atoms/Typography` is going to include major submodule with several files, but looks like no.
So, i would question if there is reason to implement it in file that name says nothing - `index.js` looks like "nothing", it's just not searchable, there are hundreds of them in workspace and IMO
I would kindly propose to name it just `src/atoms/Typography.js` and only split into several files if it grows big and unmaintainable.

TLDR: I find folder with lone `index.js` inconvenient and premature optimization.

Reply:
Ok, I understand, changed!